.pylintrc

```
@@ -0,0 +1,2 @@
+[MASTER]
+disable=fixme,missing-function-docstring,missing-module-docstring
```

We can remove these ignores later.

```suggestion
disable=fixme,missing-function-docstring,missing-module-docstring

```